src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private final int number = random.nextInt(100);
```

Comment:
`number` should be random every time you call  `getRandomBall`, so it has to be local variable in this method.